dynawo/sources/Modeler/DataInterface/DYNDataInterface.h

```diff
@@ -205,6 +205,12 @@ class DataInterface {
    * @returns cloned data interface
    */
   virtual boost::shared_ptr<DataInterface> clone() const = 0;
+
+  /**
+  * @brief test if some network components does not have a dynamic model
```

Comment:
Please explain in which case the network is instantiated or not

Reply:
ok